src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -202,15 +224,14 @@ select the input to start typing.
 
 ## Keep passwords private—but enable users to see them if they want {: #show-password }
 
-Passwords inputs should have `type="password"` to hide password text. 
-
-Using `<input type="password">` also means that browsers such as Firefox offer
-to save your password when a form is submitted (along with using the `name` and
-`id` attributes to guess the meaning of form inputs).
+Passwords inputs should have `type="password"` to hide password text and help the 
+browser understand that the input is for passwords. (Note that browsers use 
+[a variety of techniques](#autofill) in order to understand input roles and decide 
+whether or not to offer to save passwords.)
 
-However, you should add a **Show password** icon or button to enable users to
-check the text they've entered—and don't forget to add a **Forgot password**
-link. See [Enable password display](#password-display).
+You should add a **Show password** icon or button to enable users to check the 
+text they've entered—and don't forget to add a **Forgot password** link. See 
```

Comment:
```suggestion
text they're entering. Don't forget to add a **Forgot password** link. See 
```

Reply:
Not sure — this sounds a bit jumpy. Maybe just a comma, not a dash?